tests/functional/repos/test_repos.py

```diff
@@ -0,0 +1,48 @@
+from django.urls import reverse
+
+import mock
+import pytest
+
+
+class TestGetRepos:
+    @pytest.mark.django_db(transaction=True)
+    @pytest.mark.usefixtures('load_db_fixture')
+    @pytest.mark.parametrize(
+        'load_db_fixture', [{'fixture_filename': 'katka_project.json'}], indirect=True)
+    @mock.patch('requests.sessions.Session.request')
```

Comment:
I see these decorators are repeated for the two tests, do you think they will be generic for all tests? in that case, how do you feel about moving some (or all?) to the class instead?

Reply:
I have changed the class name to be more specific and moved the fixtures to class level